doc/api_methods/GetSubscriber.md

```diff
@@ -0,0 +1,89 @@
+[back to list](../Readme.md)
+
+# Get Subscriber
+
+## `array getSubscriber(string $subscriber_email)`
+
+This method throws an `\Exception` in case a subscriber with a given email address doesn’t exist.
+
+## Arguments
+| Argument | Type | Description |
+| --- | --- | --- |
+| $subscriber_email | string | a valid email address |
+
+## A subscriber data structure
+
+### Subscriber
+| Property | Type | Description |
+| --- | --- | --- |
+| id | string | Id of the subscriber |
+| wp_user_id | string | Id of a WordPress user associated with the subscriber |
+| is_woocommerce_user | string | A flag telling whether the user is also a WooCommerce customer. Possible values are: `1`, `0` |
+| first_name | string | Fist name of the subscriber. |
+| last_name | string | Last name of the subscriber. |
+| email | string | Email address of the subscriber. |
+| status | string | Status of the subscriber. Possible values are: `unconfirmed`, `subscribed`, `unsubscribed`, `bounced`, `inactive` |
+| subscribed_ip | string | An IP address used for subscription. |
+| confirmed_ip | string | An IP address used for confirmation. |
+| confirmed_at | string | UTC time of subscription confirmation in 'Y-m-d H:i:s' format |
+| created_at | string | UTC time of creation in 'Y-m-d H:i:s' format |
+| updated_at | string | UTC time of last update in 'Y-m-d H:i:s' format |
+| deleted_at | string\|null | This property in not null in case that list is in trash and contains UTC time in 'Y-m-d H:i:s' format. |
+| unconfirmed_data | string\|null | May contain serialized subscriber data in case when there are pending changes waiting for a confirmation from a subscriber |
+| source | string | Possible values: `form`,`imported`,`administrator`,`api`,`wordpress_user`,`woocommerce_user`,`woocommerce_checkout`,`unknown`)
+| unconfirmed_data | string | Counter for confirmation emails |
```

Comment:
This is a duplicate, you have already mentioned this here: https://github.com/mailpoet/mailpoet/pull/2069/files#diff-bffcc4926a39d3dfa92de8be3f98fb05R32

You probably meant: `count_confirmations` But that is not a string.

Reply:
We do return it as string :(